src/main/java/camera/Camera.java

```diff
@@ -13,51 +17,142 @@
  */
 public class Camera {
 
+    /**
+     * Class parameters.
+     */
     private List<CameraObjectDetector> cameraObjectDetectorList;
     private VideoCapture videoCapture;
     private String link;
     private Mat firstFrame;
     private Mat lastFrame = new Mat();
     private boolean changed = false;
+    private List<Mat> frameParts = new ArrayList<>();
+    private List<List<double[]>> activity;
+    private List<BackgroundSubtractorKNN> knns = new ArrayList<>();
+    private long firstTime = -1;
+    private int frameCounter = 0;
+    private final int frames = 4;
 
     /**
      * Constructor for a Camera.
+     *
      * @param newCapture The VideoCapture of this camera.
-     * @param newLink The link of this camera.
+     * @param newLink    The link of this camera.
```

Comment:
We have always just used one space for this. With this tab here, this is the only method in the project where it is different.